clang/test/Driver/uefi-data-layout.c

```diff
@@ -0,0 +1,4 @@
+// RUN: %clang -target x86_64-unknown-uefi -S -emit-llvm -o - %s | \
```

Comment:
test/Driver tests should generally only test `-###`, not `-emit-llvm -S` (codegen). Place such codegen tests to test/CodeGen